app/helpers/patients_helper.rb

```diff
@@ -1,2 +1,17 @@
 module PatientsHelper
+
+  def bootstrap_class_for flash_type
+    { success: 'alert-success', error: 'alert-danger', warning: 'alert-warning'}[flash_type.to_sym]
```

Comment:
do we need the to_sym here? I would have expected the string to have just worked...

Also, a case statement seems clearer.